docs/configuration.md

```diff
@@ -91,9 +91,11 @@ Release-it has [plenty of options][2]. See the following tables for plugin confi
 
 You can extend a configuration from a remote source using the `extends` option. The following formats are supported:
 
-- `github:owner/repo`
-- `github:owner/repo#tag`
-- `github:owner/repo:file#tag`
+- `github:owner/repo`: Get the config from the default branch (main) in the repo at Github.
+- `github:owner/repo#tag`: Get the config from the specified tag in the repo at Github.
+- `github:owner/repo:subdir#tag`: Get the config from the specified tag in the repo sub dir at Github.
+
+And support other schema, either `gitlab:`, `bitbucket:`, or `https:`.
```

Comment:
If it is HTTPS, then the file should be public and not behind authorization. However, GitHub, GitLab, and BitBucket will work as it is getting pulled with the token, right?

Reply:
Yes you right, if want to get private repo or private content from https, it could use the environment `GIGET_AUTH` to set it. The value will be set in `Authorization: Bearer ...` header

Refs to [giget docs](https://github.com/unjs/giget?tab=readme-ov-file#providing-token-for-private-repositories)

Reply:
Cool! Then I am super happy! I did not know we were working with giget, but I can see it is a transitive dependency of c12, so awesome :)